app/furniture/marketplace/stripe_events_controller.rb

```diff
@@ -24,6 +23,10 @@ def create
         order.update!(status: :paid, placed_at: DateTime.now, payment_processor_fee_cents: balance_transaction.fee)
         order.events.create(description: "Payment Received")
 
+        if marketplace.square_order_notifications_enabled?
+          order.send_to_square_seller_dashboard
```

Comment:
Thoughts on encapsulating this conditional within the method? Basically, `Order#send_to_square_seller_dashboard` is a no-op if `marketplace.square_order_notifications_enabled?` is false.